tests/emp_search.py

```diff
@@ -45,7 +46,27 @@ def test_search_by_job_title(self):
 
         for item in list_of_web_elements_jtitles:
             self.assertEqual('SDET', item.text)
-
+    def test_search_by_employee_id(self):
+        self.login('admin', 'password')
+        self.browser.find_element_by_id('empsearch_id').send_keys('635919')
+        self.browser.find_element_by_id('searchBtn').click()
+        actual_search_result = self.browser.find_element_by_link_text('635919').text
```

Comment:
Here you are finding the link by text and then asserting that the text is what you used to find it.
If you found the item by text it already means it has that text... otherwise you would not be able to find it and you would end up with NoSuchElementException - the assert is redundant?
Instead you should find the element by location - first row / 2nd cell - and then assert that it contains the expected text.